README.md

```diff
@@ -125,7 +125,7 @@ To join one of these matches check our [matchmaker documentation](https://heroic
 
 ### AI/ML model
 
-In addition to starting Nakama and database, `docker-compose.yml` file
+In addition to starting Nakama and database, `docker compose.yml` file
```

Comment:
Whooops, this was an accidental change

Reply:
Good call, no worries I'll fix it as part of an incoming PR.